apple/MarkdownParser.mm

```diff
@@ -19,7 +19,11 @@ @implementation MarkdownParser {
     const auto &markdownRuntime = expensify::livemarkdown::getMarkdownRuntime();
     jsi::Runtime &rt = markdownRuntime->getJSIRuntime();
 
+#ifdef WORKLETS_INSTALLED
+    std::shared_ptr<SerializableWorklet> markdownWorklet;
+#else
     std::shared_ptr<ShareableWorklet> markdownWorklet;
+#endif
```

Comment:
How about using a typedef in a single place rather than having `#ifdef` scattered over the whole file? Something like (not the `#ifndef` instead of `#ifdef`)
```cpp
#ifndef WORKLETS_INSTALLED
using SerializableWorklet = ShareableWorklet; // ShareableWorklet was renamed to SerializableWorklet
#endif
```
This way we can use the new type name everywhere else (for Worklets compatibility) as well as the old type name (for Reanimated v3 compatibility). Except for `extractSerializableOrThrow`.